README.md

```diff
@@ -3,7 +3,7 @@
 TEXA is a novel framework backed by a mathematical oriented theory(Texa Theory, refer to /texa-docs) for interacting, evaluating and assessing AIs.
 
 
```

Comment:
You may also add the Travis CI Badge, once you have the .travis successfully setup.
Badge code: [![Build Status](https://travis-ci.org/TexaProject/texa.svg?branch=master)](https://travis-ci.org/TexaProject/texa)